src/Components/SharedComponents/ButtonForm/ButtonForm.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, CircularProgress } from "@mui/material";
+
+interface ButtonFormProps {
+  name: string;
+  isSubmitting?: boolean;
+}
+
+export default function ButtonForm({
+  name,
+  isSubmitting = false,
+}: ButtonFormProps) {
+  return (
+    <Button
+      variant="contained"
+      fullWidth
+      type="submit"
+      disabled={isSubmitting}
+      sx={{
+        backgroundColor: "#3252DF",
```

Comment:
If we are going to implement light/dark mode functionality, I suggest sticking to the MUI theme's colors as it it has colors for both dark/light, we can also create our own theme from the design, but the first approach will be eaier and most of the colors in the design have similar ones in MUI theme.

Reply:
The sx prop has access to the theme colors with this syntax:
![image](https://github.com/user-attachments/assets/4c3ea76e-28ae-465b-9e1f-8b77088b7a4f)

Reply:
done